src/dotnet-repl/project.json

```diff
@@ -1,5 +1,4 @@
 {
-    "name": "dotnet-publish",
```

Comment:
Did you move this to the right place? It says src/dotnet-repl/project.json above.